### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or another core contributor signed this repository up for Open Collective. This pull request adds financial contributors from your Open Collective https://opencollective.com/react-native-auto-height-image ❤️

  What it does:
  - adds a badge to show the latest number of financial contributors
  - adds a banner displaying contributors to the project on GitHub
  - adds a banner displaying all individuals contributing financially on Open Collective
  - adds a section displaying all organizations contributing financially on Open Collective, with their logo and a link to their website

We have also added a `postinstall` script to let people know after `npm|yarn install` that you are welcoming donations. Feel free to remove it if you don't want it. [[More info](https://github.com/opencollective/opencollective-postinstall)]

P.S: As with any pull request, feel free to comment or suggest changes.

  Thank you for your great contribution to the Open Source community. You are awesome! 🙌
  And welcome to the Open Collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other Open Source sustainers!

### CODE REVIEW

**Comment by vivaxy** [comment]
Thanks.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Initialized by [vivaxy/gt-npm-package](https://github.com/vivaxy/gt-npm-package)
 
-[![NPM Version](http://img.shields.io/npm/v/react-native-auto-height-image.svg?style=flat-square)](https://www.npmjs.com/package/react-native-auto-height-image)
+[![Financial Contributors on Open Collective](https://opencollective.com/react-native-auto-height-image/all/badge.svg?label=financial+contributors)](https://opencollective.com/react-native-auto-height-image) [![NPM Version](http://img.shields.io/npm/v/react-native-auto-height-image.svg?style=flat-square)](https://www.npmjs.com/package/react-native-auto-height-image)
```
Comment (vivaxy) [inline README.md]: Is there a flat square badge? It's better to have a badge matching the other badge styles.

File: README.md [hunk 2]
```diff
@@ -95,3 +95,33 @@ Other image props except `resizeMode` are accepted.
 ## Contributors
```
Comment (vivaxy) [inline README.md]: Please help remove the old `Contributors` section.

File: package.json [hunk 3]
```diff
@@ -7,7 +7,8 @@
   "typings": "./index.d.ts",
   "scripts": {
     "release": "standard-version && git push --follow-tags && npm publish --registry=https://registry.npmjs.org/",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "postinstall": "opencollective-postinstall || true"
```
Comment (vivaxy) [inline package.json]: postinstall script is a little annoying for the users. Please remove it.

File: README.md (hunk 1 above)
Comment (monkeywithacupcake) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (monkeywithacupcake) [reply README.md]: updated

**Pull request merged**